minion/templates/minion-deployment.yaml

```diff
@@ -85,13 +85,23 @@ spec:
              - ALL
            {{- end }}
           {{- end }}
+          livenessProbe:
+            exec:
+              command:
+                - pgrep
```

Comment:
How about using `minion/rest/health/probe?tag=local` for liveness check. It doesn't involve any checks on broker/listeners/sink/rpc.  Just checks if bundles are resolved or not.